FrontDesk/src/FrontDesk.Blazor/Pages/AppointmentEditor.razor

```diff
@@ -0,0 +1,286 @@
+﻿@inject AppointmentTypeService AppointmentTypeService
+@inject DoctorService DoctorService
+@inject AppointmentService AppointmentService
+@inject FileService FileService
+
+<style>
```

Comment:
Use separate CSS file(s) and C# codebehind files for Blazor stuff.

Reply:
Done